File: infinity_school-python/aula11-POO-01/atividades/at02.py
'''
Crie um classe chamada pessoa com os atributos: nome, idade, peso, gênero.

'''

class Pessoa:
    def __init__(self, nome, idade, peso, genero):
        self.nome = nome
        self.idade = idade
        self.peso = peso
        self.genero = genero

pessoa1 = Pessoa("Thiago", 30, 85.5, "Masculino")
print(f"Nome: {pessoa1.nome}, Idade: {pessoa1.idade}, Peso: {pessoa1.peso} kg, Gênero: {pessoa1.genero}")

File: infinity_school-javascript/aula08-objetos_literais/at02.js
/*
Atividade 02
Crie um objeto chamado pessoa com as propriedades nome, idade, e profissao.

Em seguida:
Altere a propriedade profissao para um novo valor.
Adicione uma nova propriedade chamada cidade ao objeto.
Exiba no console o objeto completo após as modificações.

Objetivo:
Praticar a manipulação de objetos, alterando propriedades existentes, adicionando novas propriedades, e exibindo o objeto atualizado.
*/

let pessoa = {
    nome: "Thiago",
    idade: 30,
    profissao: "Engenheiro de Software"
};

pessoa.profissao = "Engenheiro de Dados";

pessoa.cidade = "Belo Horizonte";

console.log(pessoa);